src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,13 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
+  user?: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={user?.email}>
```

Comment:
```suggestion
    <a className="UserInfo" href={`mailto:${user?.email}`}>
```
https://css-tricks.com/snippets/html/mailto-links/